ext/AvizExt/viz/location_selection.jl

```diff
@@ -75,7 +75,7 @@ function ADRIA.viz.ranks_to_frequencies!(
             strokecolor=:grey,
             strokewidth=0.5,
             linestyle=:solid,
-            overdraw=true,
```

Comment:
I have a dissenting opinion.

Moving arguments around is something that's done during prototyping. Once prototyping is done, leaving the comma is untidy. If you have to adjust again (ideally months to years afterwards) then you just add the comma while prototyping.

But I think this discussion is moot because the point of having a formatting workflow is that you can do your thing and once you're finished the formatter standardises.